src/app/thankYou/thankYou.component.js

```diff
@@ -43,6 +49,14 @@ class ThankYouController{
         if(this.purchase.paymentMeans.self.type === 'elasticpath.purchases.purchase.paymentmeans'){ //only credit card type has billing address
           this.billingAddress = this.orderService.formatAddressForTemplate(this.purchase.paymentMeans['billing-address'].address);
         }
+
+        // Display Account Benefits Modal when GUEST checkout
```

Comment:
Gotcha. Interesting... Should the panel be shown if you are public also? Cuz I don't think that is currently a condition here: https://github.com/CruGlobal/give-web/pull/118/files#diff-1d741c69c93c23b65b01e053a605ecf4R22 Although maybe `registration-state` will never be completed for a public user